pkg/controller/actions/deploy/action_deploy.go

```diff
@@ -133,6 +143,56 @@ func (a *Action) run(ctx context.Context, rr *odhTypes.ReconciliationRequest) er
 	return nil
 }
 
+func (a *Action) deployCRD(
+	ctx context.Context,
+	rr *odhTypes.ReconciliationRequest,
+	obj unstructured.Unstructured,
+) (bool, error) {
+	current, lookupErr := a.lookup(ctx, rr.Client, obj)
+	if lookupErr != nil {
+		return false, fmt.Errorf("failed to lookup object %s/%s: %w", obj.GetNamespace(), obj.GetName(), lookupErr)
+	}
+
+	// the user has explicitly marked the current object as not owned by the operator, so
+	// skip any further processing
+	if current != nil && resources.GetAnnotation(current, annotations.ManagedByODHOperator) == "false" {
```

Comment:
so we expect this annoation to be added on CRD from component's manifests? i thought that is only for CR

Reply:
I don't know to be honest, it was mainly to ensure consistency regardless of the resource